internal/pkg/log/logformat.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type LogFormat int
+
+const (
+	Console LogFormat = iota
+	JSON
+)
+
+func (logFormat LogFormat) String() string {
+	return [...]string{"console", "json"}[logFormat]
+}
+
+// Creates LogFormat from string.
+// On invalid value Console is used as default with an error.
+func NewLogFormat(format string) (LogFormat, error) {
+	switch format {
+	case "console":
+		return Console, nil
+	case "json":
+		return JSON, nil
+	default:
+		return Console, errors.New(`log format must be "console" or "json"`)
```

Comment:
Not entirely sure how to handle error so I took inspiration from how you handle log file and tried to do it similarly to be consistent. But if you have a better idea please share. :)

Reply:
This line is ok, but it will probably not be needed with string type.